src/Entity/CommitteeMembership.php

```diff
@@ -98,6 +98,13 @@ class CommitteeMembership
      */
     private $enableVote;
 
+    /**
+     * @var CommitteeCandidacy|null
+     *
+     * @ORM\OneToOne(targetEntity="CommitteeCandidacy")
```

Comment:
also, a functional rule: a CommitteeCandidacy can not exist without its related CommitteeMembership

1/ tell this rule to the ORM by adding `orphanRemoval=true` to the OneToOne annotation

2/ tell this rule to the database (see my other comment with the @ORM\JoinColumn` annotation on `CommitteeCandidacy::$committeeMembership`